aRchive.py

```diff
@@ -220,7 +196,11 @@ def archive_local_repository(bioc_dir, archive_dir):
 
     BIOCONDUCTOR_DIR = os.path.abspath(args.bioconductor_dir)
     ARCHIVE_DIR = os.path.abspath(args.archive_dir)
-    print "aRchive is being run in %s " % BIOCONDUCTOR_DIR
-    print "aRchive is being stored in %s" % ARCHIVE_DIR
+    log.info("aRchive is being run in %s " % BIOCONDUCTOR_DIR)
+    log.info("aRchive is being stored in %s" % ARCHIVE_DIR)
     checkout_main_biocondutor_repository(BIOCONDUCTOR_DIR)
-    archive_local_repository(BIOCONDUCTOR_DIR, ARCHIVE_DIR)
+
+    # Make the directory which user specifies to build the archive.
+    if not os.path.exists(ARCHIVE_DIR):
+        os.mkdir(ARCHIVE_DIR)
+    archive_local_repository(os.path.join(BIOCONDUCTOR_DIR, 'Rpacks'), ARCHIVE_DIR)
```

Comment:
Not sure why you have to hardcode "Rpacks" here. I think it goes against the user specifying the bioconductor directory. 

I run it, 

`python aRchive.py ../Rpacks ../archive` 

and the program identifies the relative paths.